### PR TITLE
integration test

### PR DESCRIPTION
**Pull request by aidaiym**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: app/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -383,7 +383,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CURRENT_PROJECT_VERSION = 1.3.0;
-				DEVELOPMENT_TEAM = L8HXY9BDGG;
+				DEVELOPMENT_TEAM = "";
```
Comment (Eldar2021) [inline app/ios/Runner.xcodeproj/project.pbxproj:386]: kyzyk...

File: app/ios/Runner/GoogleService-Info.plist [hunk 2]
```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
 <dict>
 	<key>CLIENT_ID</key>
-	<string>417182149715-qokpr06kebev3u3a0lci4227kd6a6mdi.apps.googleusercontent.com</string>
+	<string>417182149715-icusnhceqetnqlinprnn9o8tamct4g7q.apps.googleusercontent.com</string>
```
Comment (Eldar2021) [inline app/ios/Runner/GoogleService-Info.plist:6]: Emne uchun?

File: app/lib/app/data/repository/auth_repositoty_impl.dart [hunk 3]
```diff
@@ -64,10 +64,10 @@ final class AuthRepositoryImpl implements AuthRepository {
...
         (r) => Right(
           UserEntity(
-            accessToken: '',
-            username: '',
-            gender: gender,
-            localeCode: '',
+            accessToken: r.accessToken,
+            username: r.username,
+            gender: r.gender,
+            localeCode: r.localeCode,
```
Comment (Eldar2021) [inline app/lib/app/data/repository/auth_repositoty_impl.dart:70]: 👍

File: app/lib/constants/api/api_const.dart [hunk 4]
```diff
@@ -9,7 +9,7 @@ class ApiConst {
 
   static const domain = 'https://myquran.life';
 
-  String get socketBase => 'wss://$_getDomain/ws';
+  String get socketBase => 'wss://myquran.life/ws';
```
Comment (Eldar2021) [inline app/lib/constants/api/api_const.dart:12]: Haha this my error. Thank you for fixing

File: app/lib/core/auth/soccial_auth.dart [hunk 5]
```diff
@@ -25,13 +26,21 @@ class SoccialAuth {
...
+        ],
+      );
+      final oauthCredential = OAuthProvider('apple.com').credential(
+        idToken: appleCredential.identityToken,
+        accessToken: appleCredential.authorizationCode,
+      );
+      final userCredential = FirebaseAuth.instance.signInWithCredential(oauthCredential);
+      return userCredential;
+    } catch (e) {
+      rethrow;
```
Comment (Eldar2021) [inline app/lib/core/auth/soccial_auth.dart:43]: Cool!

**Review by Eldar2021 — COMMENTED** [review]
1) Why our apple team id removed?
2) Why our client id changed?

File: app/ios/Runner/GoogleService-Info.plist (hunk 2 above)
Comment (aidaiym) [reply app/ios/Runner/GoogleService-Info.plist:6]: firebase'те акыркы info plist' көчүргөндө ушундай болду

File: app/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (aidaiym) [reply app/ios/Runner.xcodeproj/project.pbxproj:386]: буну билбейт экем

**Review by Eldar2021 — APPROVED** [review]
I think we can merge it

**Pull request merged**